### ISSUE TITLE
[Visual requirement-Command Palette]-Luminosity ratio is 4.046:1 which is less than 4.5:1 for the shortcut text appended to the Commands.

### ISSUE THREAD

**Issue by DasaradhaKamjula** [microsoft/terminal#7915]
**User Experience:**
Vision impairment users will not be able to view the text clearly if the Luminosity ratio is less than 4.5:1 .

**Test Environment:**
App: Windows Terminal Preview
Feature: Command Palette
OS: Win 10 2004 Build (19041.508)

**Repro Steps:**
1.Open Windows terminal Preview app
2.Open Command Palette using shortcut (Ctrl+shift+p)
3.verify the Luminosity ratio for shortcut text.

**Actual:**
Luminosity ratio is 4.046:1 which is less than 4.5:1 for the shortcut texts appended to the commands.(For Example: Luminosity ratio of Close pane command shortcut text Ctrl+Shift+W is 4.04:1 )

**Expected:**
Luminosity ratio should be greater than 4.5:1 for the short cut texts which are appended to the commands

**Comment by zadjii-msft** [comment]
Yep this is reasonable. I bet we could make the background of the keys even _darker_ (in dark mode) relative to the cmdpal background, and that'll probably look good.

**Comment by Hegunumo** [comment]
I’m going to work on this

**Comment by DHowett** [comment]
This issue was fixed in #7937 and deployed to production in November.

**Comment by zadjii-msft** [comment]
@awalia13 / @DasaradhaKamjula If this is confirmed as fixed, can we close this issue now?

**Comment by zadjii-msft** [comment]
@Chandresh1007  If this is confirmed as fixed, can we close this issue now?

**Comment by Priyanka-Chauhan123** [comment]
Issue is fixed. Verified on Windows terminal version: 1.8.1521.0 and OS version: Win 10 build 21390.2025. Hence, closing the bug.
